src/modules/contacts/pages/ContactOverviewPage.vue

```diff
@@ -17,34 +17,196 @@
     </div>
     <div>
       <AppHeading>{{ t('contactOverview.information') }}</AppHeading>
+      <AppInfoList>
+        <AppInfoListItem
+          :name="'Preferred channel'"
+          :value="contact.profile.preferredContact"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.email')"
+          :value="contact.email"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.phone')"
+          :value="contact.profile.telephone"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.deliveryOptIn')"
+          :value="contact.profile.deliveryOptIn"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.newsletter')"
+          :value="contact.profile.newsletterStatus"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.groups')"
+          :value="contact.profile.newsletterGroups.join(',')"
```

Comment:
Join with `', '` (a space after the comma) for slightly nicer formatting?

Reply:
Done: https://github.com/beabee-communityrm/beabee-frontend/pull/104/commits/578f343a4e4679ed35823e36a989aa613b9789b8